complete/bookstore/pom.xml

```diff
@@ -31,6 +31,14 @@
       <artifactId>spring-boot-starter-test</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
This should not be needed `spring-boot-starter-test` should include junit.  It does not make sense also that it would only be needed for maven and not for gradle

Reply:
ok, done